src/components/WishedCard.jsx

```diff
@@ -0,0 +1,73 @@
+import { Text, View, TouchableOpacity, Image } from 'react-native';
+import React, { Component } from 'react';
+import { FontAwesome, Ionicons } from '@expo/vector-icons';
+import { AirbnbRating } from 'react-native-ratings';
+import { COLORS } from '../styles/colors';
+import { containers } from '../styles/HomeScreen/Components_ProductCard';
+import { elements } from '../styles/HomeScreen/Components_ProductCard';
+
+export default class WishedCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isFavorite: false,
+    };
+  }
+  render() {
+    return (
+      <View style={containers.mainContainer}>
+        <TouchableOpacity
+        // onPress={() => {
+        //   this.props.navigation.navigate('ProductDetails', {
+        //     product: this.props.product,
+        //   });
+        // }}
```

Comment:
Do you need this lines?

Reply:
We'll be using that to test the product detail screen when it's done, meanwhile it is just a placeholder that we might delete if we decide we won't need it.